src/braket/device_schema/jaqcd_device_action_properties.py

```diff
@@ -68,10 +69,12 @@ class JaqcdDeviceActionProperties(DeviceActionProperties):
         ...         "minShots": 0,
         ...         "maxShots": 4,
         ...     }],
+        ...    "noQubitRewiringSupported": True
         ... }
         >>> JaqcdDeviceActionProperties.parse_raw(json.dumps(input_json))
 
     """
 
     supportedOperations: List[str]
     supportedResultTypes: Optional[List[ResultType]]
+    noQubitRewiringSupported: Optional[bool] = None
```

Comment:
To confirm, would we always expect this to be a binary decision and never anything in between (partial rewiring with constraints etc) ? Otherwise, structuring this as an enum might be a more extensible approach.

Reply:
Yeah, `noQubitRewiring` is confusing; `disableQubitRewiring` might be better.

I've thought a lot about having multiple options for qubit rewiring, and had settled on this because running with explicit allocation is an especially important use case. As for other options, that opens up a new discussion about compilation options as a whole. That said, I'm open to the idea of having a `qubitRewiringStrategy` field instead, which would be an enum.

Reply:
I agree `disableQubitRewiring` is better. 

I also think that it still leaves open the possibility of having a `qubitRewiringStrategy` in the future as well.